terraform/lamp-stack-ibmcloud/scripts/setup_env.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -xe 
```

Comment:
Can we get a high level header about what this file does?

Reply:
Will do.